crates/router/src/core/payments/helpers.rs

```diff
@@ -4417,6 +4490,98 @@ pub async fn get_additional_payment_data(
                 details: Some(open_banking.to_owned().into()),
             },
         )),
+        domain::PaymentMethodData::CardDetailsForNetworkTransactionId(card_data) => {
+            let card_isin = Some(card_data.card_number.get_card_isin());
+            let enable_extended_bin =db
```

Comment:
this is a new config we are adding? If this a new config then instead using config, profile. Add a column in profiles

Reply:
No, this is not the new config. This was the config introduced in order merchants to block payments using this field [PR](https://github.com/juspay/hyperswitch/pull/3312).